ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -23,288 +22,246 @@ module('Acceptance | sync | overview', function (hooks) {
     this.permissions = this.owner.lookup('service:permissions');
+    this.version.type = 'enterprise';
   });
 
-  module('ent', function (hooks) {
-    hooks.beforeEach(async function () {
-      this.version.type = 'enterprise';
-    });
-
-    module('sync on license', function (hooks) {
-      hooks.beforeEach(async function () {
-        this.version.features = ['Secrets Sync'];
-      });
-
-      module('when feature is activated', function (hooks) {
-        hooks.beforeEach(async function () {
-          syncHandlers(this.server);
-          await authPage.login();
-        });
+  test('it hides sync overview on community', async function (assert) {
+    this.version.type = 'community';
+    await authPage.login();
+    assert.dom('[data-test-sidebar-nav-panel="Cluster"]').exists();
+    assert.dom(GENERAL.navLink('Secrets Sync')).doesNotExist('Secrets sync nav link does not exist');
+  });
 
-        test('it fetches destinations and associations', async function (assert) {
-          assert.expect(2);
+  test('it hides sync overview for users without policy access', async function (assert) {
+    this.server.get('/sys/internal/ui/resultant-acl', () => {
+      return {
+        data: {
+          exact_paths: {},
+          glob_paths: {
+            'sys/sync/': { capabilities: ['deny'] },
+          },
+        },
+      };
+    });
+    await authPage.login();
+    assert.dom('[data-test-sidebar-nav-panel="Cluster"]').exists();
+    assert.dom(GENERAL.navLink('Secrets Sync')).doesNotExist();
+  });
 
-          this.server.get('/sys/sync/destinations', () => {
-            assert.true(true, 'destinations is called');
-          });
-          this.server.get('/sys/sync/associations', () => {
-            assert.true(true, 'associations is called');
-          });
+  test('it hides sync overview if not on ent license', async function (assert) {
+    this.server.get('/sys/license/features', () => {
+      return { features: [] };
+    });
+    await authPage.login();
+    assert.dom('[data-test-sidebar-nav-panel="Cluster"]').exists();
+    assert.dom(GENERAL.navLink('Secrets Sync')).doesNotExist('Secrets sync nav link does not exist');
+  });
 
-          await visit('/vault/sync/secrets/overview');
-        });
+  test('it navigates to sync overview for users with policy access', async function (assert) {
+    this.server.get('/sys/internal/ui/resultant-acl', () => {
+      return {
+        data: {
+          exact_paths: {},
+          glob_paths: {
+            'sys/sync/': { capabilities: ['create', 'read', 'update'] },
+          },
+        },
+      };
+    });
+    await authPage.login();
+    assert.dom('[data-test-sidebar-nav-panel="Cluster"]').exists();
+    await click(GENERAL.navLink('Secrets Sync'));
+    assert.strictEqual(currentURL(), '/vault/sync/secrets/overview', 'it navigates to overview');
+  });
 
-        module('when there are pre-existing destinations', function (hooks) {
```

Comment:
this module housed only one test, so moved the `syncScenario` to that test directly